ztp/policygenerator/policyGen/policyHelper.go

```diff
@@ -30,19 +30,68 @@ func CreatePolicySubject(policyName string) utils.Subject {
 	return subject
 }
 
-func CreatePlacementRule(name string, namespace string, matchKeyValue map[string]string) utils.PlacementRule {
+/* Func CheckBindindRules checks the following invalid rules:
+----------------------
+bindingRules:
+	labelKey: ""
+bindingExcludedRules:
+	labelKey: ""
+-----------------------
+bindingRules:
+	labelKey: "labelValue"
+bindingExcludedRules:
+	labelKey: ""
```

Comment:
Is the reverse valid?
binding:
  labelKey: ""
exclude
  labelKey: "abcd"

Reply:
Yes, this is valid. It checks for the clusters has label labelKey but its value not in abcd.